README.md

```diff
@@ -15,6 +15,11 @@ Type in the following in a terminal:
 ```bash
 git clone https://github.com/ozone-his/ozone-docker
 cd ozone-docker
+# Create the web Docker network if it does not exist yet.
+# The project is configured to run behind Traefik running in the web Docker network but the demo uses Apache.
+# So you have to create this network for Docker Compose to work.
+# For running behind Traefik see the manual instructions.
```

Comment:
This is too verbose for a quick start, I would suggest not writing any of this and just make sure that we come up with a compact set of instructions that work for everyone.

Such details need to be provided in the manual setup guide.